.vscode/settings.json

```diff
@@ -2,7 +2,7 @@
 	"editor.defaultFormatter": "esbenp.prettier-vscode",
 	"editor.formatOnSave": true,
 	"editor.codeActionsOnSave": {
-		"source.fixAll": true
+		"source.fixAll": "explicit"
```

Comment:
I think you can change this file to the original version so that it will not be included in this PR. 
Hint: since you're only changing files in "packages" folder, doing `git add packages` will prevent adding other unnecessary files for you.